gokart/target.py

```diff
@@ -91,9 +93,14 @@ def _get_task_lock_params(self) -> TaskLockParams:
 
     def _load(self) -> Any:
         with self._target.open('r') as f:
-            return self._processor.load(f)
+            obj = self._processor.load(f)
+            self._validator(obj)
```

Comment:
@ujiuji1259 You can use retrun value of validator and let this process fail if return value is False